site/docs/contribute.md

```diff
@@ -388,6 +388,34 @@ When passing boolean arguments to existing or external methods, use inline comme
   dropTable(identifier, purge);
 ```
 
+#### Accessing instance variables
```

Comment:
Curious whether there is a format rule for this.

Reply:
I wanted to look into ways to automate this after documenting. I took a look now. It doesn't seem there is an existing checkstyle rule that would enforce the behavior we are looking for. We may try doing something with regular expressions but it may be hard to capture the context.